src/mobile/src/ui/views/onboarding/EnterSeed.js

```diff
@@ -85,6 +85,7 @@ class EnterSeed extends React.Component {
         this.state = {
             seed: '',
             isModalVisible: false,
+            isIPhone11: isIPhone11(),
```

Comment:
I don't think this should be set to state. We should be able to just use it directly.